### DISCUSSION TITLE
Добавление модели.

### DISCUSSION THREAD

**Discussion by erigs** [Arkady23/V380-Web#14] [General]
Доброго здоровья!

хотелось бы добавить следующую модельку:
{"soft_id":"JL_A9C","dev_id":43181198,"version":1000401,"timestamp":1590970124}

не совсем понятно из описания, как самому собрать. Думаю, у Вас получиться это быстрее. 
СПАСИБО!
куплена тут: https://aliexpress.ru/item/4001089979397.html

С уважением,
Игорь.

**Comment by Arkady23** [comment]
Игорь, попробуйте этот установщик 
[ark-add-on.zip](https://github.com/Arkady23/V380-Web/files/6948917/ark-add-on.zip)
Номер версии: 9090909, Soft_id: JL_A9C. Если номер версии не схватится, то сделаем: 1000402.

**Comment by Arkady23** [comment]
Все патчи обновлены. Если не смогли установить web-интерфейс, пробуйте новый патч.

**Comment by Arkady23** [comment]
Добавил описание для установки httpd вручную для новых камер.
